### ISSUE TITLE
Enhance multiparty documentation

### ISSUE THREAD

**Issue by EnriqueL8** [hyperledger/firefly#1475]
Both @SamMayWork and @Chengxuan raised in the last community call that setting up a multiparty setup using the Firefly CLI or standalone can be complicated and frustrating. Sam has gone through this process recently. 

Due to this friction, we agreed on the call to enhance the documentation to help users setup a multiparty network. There are number of discord threads that could be used as a guideline of what we want to cover in the documentation:
- https://discord.com/channels/905194001349627914/1197628222028906696/1199467976093597786 - Deploying the multiparty contract 
- https://discord.com/channels/905194001349627914/928377875827154984/1184368448881557504 - Disabling multiparty in the helm charts

And I think a general tutorial on how to set it up

**Comment by icemagno** [comment]
I've been waiting for this for centuries. I've been trying to configure my network for 3 months with the following objectives:

1. Create two organizations on separate servers, but sharing an existing and 100% functional Besu blockchain, as well as an existing and functional IPFS network.

2. Configure Data Exchange for this scenario.

3. After having this functional environment, create another organization/node on a third server and have it join the already created environment, gaining knowledge to replicate the process as many times as necessary.

I'm stuck now in how to create a new organization, since I already done with Besu and IPFS. I don't know how to make another Org/Node because the DataExchange key and cert and the ```/api/v1/network/organizations``` POST endpoint is very poor in examples in what I need to do before call it ( who will handle the dataexchange certificates distribution ? me ? )

So.. Need I to take the new certificate and put it in all dataexchange cert-peers folder and restart them all? 

How to deal with peering? 

Many many thanks to give a light on this question.

**Comment by EnriqueL8** [comment]
Yes, definitely a lot of content that is missing and now should be easier to automate and configure with the Helm Chart https://github.com/hyperledger/firefly-helm-charts
